### ISSUE TITLE
Infinite Depth

### ISSUE THREAD

**Issue by HardikJain02** [YvanYin/Metric3D#145]
Room having bright windows tends to have infinite depth, hence non-desirable results. Any workaround?

**Comment by JUGGHM** [comment]
> Room having bright windows tends to have infinite depth, hence non-desirable results. Any workaround?

One way is to first adjust the contrast and brightness of the images. Additionally, consider filtering out regions based on predicted confidence map. 

I think it might be challenging to estimate the depth of over-exposed pixels.

**Comment by HardikJain02** [comment]
Yeah, brightness might work but for my use case, I cannot prefer it.

I attempted to adjust the depth of the window to match that of the wall it's attached to. I believe there's an issue with the axis transformation. I'll work on fixing it and will update you if it resolves the problem.

**Comment by Nacriema** [comment]
Hi, did you solved the problem ?
